### ISSUE TITLE
Mapping rules to WCAG 2.0 Failure Techniques is problematic.

### ISSUE THREAD

**Issue by moekraft** [w3c/wcag-act#15]
The part about “ACT Rules should map to [WCAG 2.0 Failure Techniques]” is problematic because we have relatively few of those. - Bruce Bailey

[WCAG WG Survey - November 2016](https://www.w3.org/2002/09/wbs/35422/ACT_DPUB_Review_Nov_2016/results#xq10)

**Comment by moekraft** [comment]
Can't reconcile "ACT Rules should map to [WCAG 2.0 Failure Techniques]" and "It does not include any ACT Rules". Either it maps rules or it doesn't include rules, not both. - Michael Cooer

**Comment by moekraft** [comment]
The document says (section 3.8 2.8) that the rule set will test for failures. Will these rules be based solely on our documented failure techniques? If so the rule set may be limited. Some people have given up trying to introduce failures. 

David has pointed out [1] that the WCAG Working Group has approved 3 small administrative failures in 8 years. 

John Foliot has said [2] "I also struggle myself to understand how/why Failure Techniques are so important to some content creators. In a simplistic view, Failures seem to be 'Don’t do this' statements, whereas Techniques for Success are 'You should do this (or this, or this, or this)' – and so I see the fundamental difference as being one between positive re-enforcement and negative re-enforcement, and as the guy known to state 'Be the Fireman and not the Cop' I am struggling to understand how the negative re-enforcement provides value (but I leave open the possibility that it does, and I’m just not seeing it)."

Will the Working Group will be adding further failure techniques? 

[1] https://lists.w3.org/Archives/Public/w3c-wai-gl/2016AprJun/0336.html
[2] https://lists.w3.org/Archives/Public/w3c-wai-gl/2016AprJun/0340.html 

- Laura Carlson

**Comment by moekraft** [comment]
@Laura asked (quoting JF): “I am struggling to understand how the negative re-enforcement provides value”
Failure techniques are not really for content authors, but for end-user who are empowered by being able to point to them when complaining about inaccessible websites. They are very powerful. - Bruce Bailey
